### ISSUE TITLE
Unknown (and unused) toggle button on brave://settings/fonts

### ISSUE THREAD

**Issue by A-t-s-u-s-h-i** [brave/brave-browser#38032]
## Description 

There are 2 unknown (and unused?) toggle buttons on `brave://settings/fonts`.

## Steps to Reproduce
   1. Open `brave://settings/fonts`

## Actual result:

<img width="834" alt="スクリーンショット 2024-05-02 19 05 19" src="https://github.com/brave/brave-browser/assets/79502386/618ad5b3-19d6-407a-a077-6de417a3a913">

## Expected result:
These toggle buttons shouldn't be there.

## Brave version (brave://version info)

Desktop v1.65.126

## Version/Channel Information:

- Can you reproduce this issue with the current release? 
Yes

- Can you reproduce this issue with the beta channel? 
No

- Can you reproduce this issue with the nightly channel? 
No

## Other Additional Information:

- Is the issue reproducible on the latest version of Chrome? 
No

**Comment by LaurenWags** [comment]
Reproduces on `1.65.126` (Release) for me also. Does not reproduce on `1.67.52` (Nightly) for me.

**Comment by rebron** [comment]
<img width="718" alt="Screenshot 2024-05-10 at 11 46 43 AM" src="https://github.com/brave/brave-browser/assets/5951041/482ea3fb-7b8a-4328-8748-ae7febf5bcdf">
Looks like we'll have this fixed in 1.66. Checked with 1.66.104

**Comment by kjozwiak** [comment]
@rebron if that's the case, we should close the above and move it into `1.66.x` so it can be verified by @brave/qa-team 👍

**Comment by rebron** [comment]
Closing as resolved in 1.66.x.

**Comment by GeetaSarvadnya** [comment]
Verification **PASSED** on

```
![image](https://github.com/brave/brave-browser/assets/38657976/969283fd-4582-435a-906f-b56d54949b60)

```
![image](https://github.com/brave/brave-browser/assets/38657976/2e6a9e16-04cd-412a-a6d5-060bf9381577)

Reproduced the issue on `1.65126` for Japan locale. The issue is not reproducible for other locales.
![image](https://github.com/brave/brave-browser/assets/38657976/37ff132a-af59-4f0f-9d2e-b52bbdfe12cf)

The issue is fixed and working fine on `1.66.108`
![image](https://github.com/brave/brave-browser/assets/38657976/0c41493a-17ca-459e-9ff9-e36d663ec1c1)
